tools/cache_creator/test/RequiredCacheCreatorArguments.spvasm

```diff
@@ -0,0 +1,28 @@
+; Check that cache-creator requires all the necessary arguments to create a valid cache file.
+; These tests doesn't require SPIR-V input, but we use .spvasm as the file extension so that LIT can pick it up.
+
+; Test 1: Empty input ELF list.
+; RUN: not cache-creator -o %t.bin --uuid=00000000-0000-0000-0000-000000000000 --device-id=0x6080 2>&1 | FileCheck --check-prefix=CHECK-ELFS %s
+; CHECK-ELFS:      Not enough positional command line arguments specified!
+; CHECK-ELFS-NEXT: Must specify at least 1 positional argument
```

Comment:
This is a poor error message.  How are people suppose to know that the elf files are the positional arguments.

Reply:
This error is auto-generated by the llvm command line library, and you would see the same error message with any llvm tool that accepts multiple inputs AFAIK. We could work around it by allowing zero inputs and erroring out manually, but the very next line of the error message that's part of the test tells you to run with `--help` to see usage details. I'd rather leave this as-is.
What do you think?

Reply:
It is not idea, but it should be fine for now.  Even if this is common for llvm command line tools, that does not mean all users will be familiar with it.  However, as you pointed out the "--help" is available which should be enough.